Source/WebCore/platform/graphics/cocoa/VP9UtilitiesCocoa.mm

```diff
@@ -124,6 +124,9 @@ void registerSupplementalVP9Decoder()
 
 bool isVP9DecoderAvailable()
 {
+    if (auto disabledForTesting = VP9TestingOverrides::singleton().hardwareDecoderDisabled())
```

Comment:
we are changing the purpose of this method and it now no longer does what the name indicates.
`setHardwareVP9DecoderDisabledForTesting` is in effect `setVP9DecoderDisabledForTesting`

A new method should be added that disable the VP9 decoder rather than modify/hack the behaviour of an existing method to suit a different purpose.

In the mean time, I'm happy to just drop the 
```
                if (window.internals) {
                     // Disable VP9 for testing on macOS to confirm VP8 still works.
                     internals.setHardwareVP9DecoderDisabledForTesting(true);
                 }
```

Reply:
Ok, will add a second internal method specifically for disabling the hardware VP9 decoder and use that directly, to minimize disruption.